src/styles/blocks/header.scss

```diff
@@ -0,0 +1,58 @@
+.header {
+  width: 100%;
+  padding: 0 50px 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__img {
+    display: flex;
+  }
+}
+
+.nav {
```

Comment:
each BEM block should be in the separate block

Reply:
like it is, or what do you mean with second block?

Reply:
for now main quastion for the test because i can't pass it